src/coreimage.cs

```diff
@@ -4729,70 +4733,76 @@ interface CIAttributedTextImageGenerator {
 	[TV (11,0)]
 	[BaseType (typeof (CIFilter))]
 	interface CIBarcodeGenerator {
-		// TODO: Needs review
-		//[CoreImageProperty ("inputBarcodeDescriptor")]
-		//CIBarcodeDescriptor BarcodeDescriptor { get; set; }
+		[CoreImageFilterProperty ("inputBarcodeDescriptor")]
+		CIBarcodeDescriptor BarcodeDescriptor { get; set; }
 	}
 
 	[CoreImageFilter]
 	[iOS (11,0)]
 	[Mac (10,13)]
 	[TV (11,0)]
-	[BaseType (typeof (CIFilter))]
+	[BaseType (typeof (CIFilter))] // Maybe 'CIScaleTransform' (shared 'Scale' and 'AspectRatio' property)?
```

Comment:
does `CIScaleTransform` exists ?
it's possible to add our own - but it can bite us back in the future if Apple introduce the same with different properties.

Reply:
I attached generated code for the `SuperClass` in https://github.com/xamarin/xamarin-macios/pull/2974#issuecomment-343296118, `CIScaleTransform` isn't there so we'd have to create our own and yes that could potentially bite us which is why I added the comment :P